bool face_exist
string name
float64 confidence